src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -32,12 +32,27 @@ public RaspberryPi3Driver()
         {
             if (Environment.OSVersion.Platform == PlatformID.Unix)
             {
-                _internalDriver = new RaspberryPi3LinuxDriver();
-                RaspberryPi3LinuxDriver linuxDriver = _internalDriver as RaspberryPi3LinuxDriver;
+                RaspberryBoardIdentification identification = RaspberryBoardIdentification.LoadBoard();
+                RaspberryPi3LinuxDriver linuxDriver = null;

```

Comment:
I don't believe the initialization with null is needed since you cover all paths